### PR TITLE
Refactoring Routes

### PR DESCRIPTION
**Pull request by Sent1nelDNoir**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPYTTTN-43)

## Code reviewers

- [ ] @antooa
- [ ] @HalynaDom
- [ ] @OlexaNdrus
- [ ] @pinkfoxxx
- [ ] @Rocckk
- [ ] @Rostyk1404
- [ ] @Svviat
- [ ] @ValentynLiashenko

### Second Level Review

- [ ] @Blackleaves
- [ ] @yevgenbehen
- [ ] @lhalam

## Summary of issue
Refactoring Routes

### CODE REVIEW

File: iBudget/static/src/router/routes.js [hunk 1]
```diff
@@ -24,49 +32,47 @@ export default [
...
-        path: '/add_spending', component: Icon_getter
+        path: '/spendings/limit_ind', component: Spend
     },
     {
-        path: '/spending_add', component: Spending_add
+        path: '/spendings/limit_group', component: Limit
     },
     {
-        path: '/Financial_goal', component:Financial_goal
+        path: '/spendings/new', component: Spending_add
```
Comment (Blackleaves) [inline iBudget/static/src/router/routes.js]: What is difference between `/spendings/add` and `/spendings/new` ?

File: iBudget/static/src/router/routes.js [hunk 2]
```diff
@@ -24,49 +32,47 @@ export default [
...
-        path: '/password_recovery/:token', component: PasswordRecovery
-    },
-    {
-        path: '/Income_registration', component: Income_registration
+        path: '/incomes/tracking', component: Income_tracker
     },
+
     {
-        path: '/goal', component: Goal
+        path: '/incomes/add', component: Income_registration
```
Comment (Blackleaves) [inline iBudget/static/src/router/routes.js]: What is deifference between `/incomes/add` and `/incomes/new` ?

Comment (Sent1nelDNoir) [reply iBudget/static/src/router/routes.js]: In the future we plan to leave one button with name "add". which will allow the user to add an entry to the history and create a new category( this one for income). Now we have separate buttons  one for  create new category (new) an another one for make entry in income history(add).

File: iBudget/static/src/router/routes.js (hunk 1 above)
Comment (Sent1nelDNoir) [reply iBudget/static/src/router/routes.js]: In the future we plan to leave one button with name "add". Цhich will allow the user to add an entry to the history and create a new category( this one for spending). Now we have separate buttons  one for  create new category (new) an another one for make entry in spending history(add).

File: iBudget/static/src/router/routes.js (hunk 2 above)
Comment (Sent1nelDNoir) [reply iBudget/static/src/router/routes.js]: Maybe  better put the plus icon on this button .

File: iBudget/static/src/router/routes.js (hunk 1 above)
Comment (Sent1nelDNoir) [reply iBudget/static/src/router/routes.js]: Maybe better put the minus icon on this button.

**Pull request merged**